examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,470 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eca904da",
+   "metadata": {},
+   "source": [
+    "## Packages and imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "93cdcd4d",
+   "metadata": {},
+   "source": [
+    "We'll begin by installing the relevant analytics packages that we'll use for our examples. We'll need:\n",
+    "* `arrow`\n",
+    "* `ggplot2`\n",
+    "* `jsonlite`\n",
+    "* `geojsonio`\n",
+    "* `sf`\n",
+    "* `tidyverse`\n",
+    "\n",
+    "which you can install individually by running `install.packages(arrow)`, etc. These packages allow us to load parquet data, interact with and format data the way we want, and visualize our result. Additionally, we'll need two packages specific to accessing Microsoft's Azure platform: `AzureStor` and `AzureRMR`.\n",
+    "\n",
+    "While common practice is to use the `rgdal` package for spatial analytics purposes (see last example), `rgdal` will reach its end-of-supported lifecycle in early 2023. Hence, we recommend the new industry standard `sf` package (standing for \"simple features\")."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f2e39b1c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import relevant packages\n",
+    "library(arrow)\n",
+    "library(jsonlite)\n",
+    "library(AzureStor)\n",
+    "library(AzureRMR)\n",
+    "library(sf)\n",
+    "library(geojsonio)\n",
+    "library(tidyverse)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "acaf7238",
+   "metadata": {},
+   "source": [
+    "## Access and download data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e0871e7a",
+   "metadata": {},
+   "source": [
+    "Now that we've got our package environment set up, let's load the data. This notebook demonstrates the important fact that the output of the PHDI Building Blocks pipeline are cloud-hosted files that are accessible locally for all manner of analytics. Let's access the Storage Account and pull down the parquet data to read into R."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d76d4d85",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Configure container access to Azure storage\n",
+    "az <- create_azure_login(tenant=\"9ce70869-60db-44fd-abe8-d2767077fc8f\", host=\"https://storage.azure.com/\")\n",
+    "token <- az$token$credentials$access_token\n",
+    "adls_ep <- adls_endpoint(\"https://pitestdatasa.dfs.core.windows.net/\", token=token)\n",
```

Comment:
Keep pitestdatasa implementation-specific value out of source control.

Reply:
As above